packages/renderer/src/stores/event-store.ts

```diff
@@ -20,6 +20,7 @@ import humanizeDuration from 'humanize-duration';
 import type { Component } from 'svelte';
 import type { Writable } from 'svelte/store';
 
+import { isConfigurationChangedArg } from '../../../main/src/plugin/configuration-impl';
```

Comment:
this is a problem; we cannot import function from the main package into the renderer

Reply:
@benoitf What is the recommended way to do this? 

I can see this pattern used a few times (for example https://github.com/podman-desktop/podman-desktop/blob/f99f2ab1c5ea845ad0a7101a837b7b8022ca2184/packages/renderer/src/stores/recommendedRegistries.ts#L21) and there is no `shared` package

Reply:
You can move the function to the renderer package, if this need to be in both, should be duplicated

Reply:
Thanks. This is what I finally done, see latest commit